server/services/personalPageService.ts

```diff
@@ -216,6 +216,11 @@ export default class PersonalPageService {
       }
     }
 
+    let nationality = 'Not entered'
+    if (inmateDetail?.profileInformation?.some(entry => entry.type === 'NAT')) {
+      nationality = inmateDetail.profileInformation.filter(entry => entry.type === 'NAT')[0].resultValue
+    }
```

Comment:
Maybe this could be:

```
const nationality = inmateDetail?.profileInformation
    ?.find(entry => entry.type === 'NAT')
    ?.resultValue 
    || 'Not entered'
```